.gitignore

```diff
@@ -33,6 +33,9 @@ qa/.vm_ssh_config
 qa/.vagrant
 qa/acceptance/.vagrant
 qa/Gemfile.lock
+Gemfile.jruby-1.9.lock
+.lock
```

Comment:
@ph I had this being left over once or twice already(likely from dirty killing a build).  No reason not to ignore it right, we don't ever want to commit that? :)